Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,8 +25,13 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 ```json
 {
     "runtimeOptions": {
-        "gcServer": true,
-        "gcConcurrent": false,
+
+        "System.GC.Server": true,
+        "System.GC.Concurrent": false,
+        "System.Threading.ThreadPool.MinThreads": 4,
+        "System.Threading.ThreadPool.MaxThreads": 8,
+        "System.Threading.Thread.UseAllCpuGroups": true
+
         "framework": {
             "name": "Microsoft.DotNetCore",
```

Comment:
I think ToString would be fine, the component the config applies to can choose whether to ignore or how to issue an error.

Also, since we plan on exposing these properties eventually, is there a plan to remove reference to "runtime"? I think a "properties" section is fine instead "runtimeOptions", but what about the file name?